spec/requests/api/v1/user_requests_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'User Requests' do
+  context 'POST /api/v1/users' do
+    it 'should create a user and return a 204 if successful' do
+      body = { uid: 123456 }.to_json
```

Comment:
I'm thinking we'll want more info when creating a user, is this correct?

Reply:
Good question. I was wondering if that would be more handled with local storage on the frontend. We can extend the attributes if we decide to gather more info as part of registration. What do you think?

Reply:
I think we'll definitely need at least the user name of the users, mostly because displaying the name on other peoples comments will be impossible if we rely on local storage. We should also bring in the email so we can maybe have gravatar support later on. what do you think?

Reply:
Firebase sends back a URL for a users GitHub profile pic.

Reply:
Nice, then I imagine we should store that url as well, as long as it doesn't change.

Reply:
Sounds good to me.

Reply:
ok awesome, I'll merge this in then when those fields are added to the db

Reply:
Ok, so here's what I'm gonna change:

1. UID needs to be a string, not an integer
2. I'm going to add the following fields that will be populated from the auth info:
  - `email` `string`
  - `username` `string`
  - `profile_picture_url` `string`